internal/version/version.go

```diff
@@ -110,25 +110,21 @@ func (v PatchVersion) String() string {
 
 // ToMinor satisfies Version.ToMinor
 func (v PatchVersion) ToMinor() MinorVersion {
-	splitDebug := strings.Split(string(v), debugSuffix)
```

Comment:
this is unrelated, but I fixed the only place in this file we didn't use the regex